### ISSUE TITLE
Error: spawn cordova ENOENT 

### ISSUE THREAD

**Issue by devnigelwheeler** [ruddell/generator-jhipster-ionic#26]
Hi, I am getting the following error when running the generator after selecting a jhipster project folder:

```
Which application do you want to generate from? rbfjhio                                                                                                                                                                       
events.js:160                                                                                                                                                                                                                   
      throw er; // Unhandled 'error' event                                                                                                                                                                                      
      ^                                                                                                                                                                                                                         
                                                                                                                                                                                                                                
Error: spawn cordova ENOENT                                                                                                                                                                                                     
    at exports._errnoException (util.js:1036:11)                                                                                                                                                                                
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)                                                                                                                                                   
    at onErrorNT (internal/child_process.js:359:16)                                                                                                                                                                             
    at _combinedTickCallback (internal/process/next_tick.js:74:11)                                                                                                                                                              
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by ruddell** [comment]
Do you have Cordova installed?  I need to add it to the installation instructions - should be fixed soon via #26

**Comment by devnigelwheeler** [comment]
Did so via npm, plus generator-m-ionic and all seems to run. Many thanks.
